### PR TITLE
fix next button in pagination disappears on certain page

### PR DESCRIPTION
**Pull request by 0xF013**
[`* `#15598 ` Next button in pagination disappears on certain page`](http://www.openproject.org/work_packages/15598)

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://0xf013-feature-15598-next-button-in-pagination-disappears-on-ce.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

**Comment by 0xF013** [comment]
[`* `#15605 ` pagination not updated when switching from a non-empty to an empty result list`](http://www.openproject.org/work_packages/15605)

File: app/assets/javascripts/angular/directives/components/table-pagination-directive.js [hunk 1]
```diff
@@ -67,6 +67,8 @@ angular.module('openproject.uiComponents')
       function updateCurrentRangeLabel() {
         if (scope.totalEntries){
           scope.currentRange = "(" + PaginationService.getLowerPageBound() + " - " + PaginationService.getUpperPageBound(scope.totalEntries) + "/" + scope.totalEntries + ")";
+        } else {
+          scope.currentRange = "(0 - 0/0)";
```
Comment (myabc) [inline app/assets/javascripts/angular/directives/components/table-pagination-directive.js:71]: I actually think we should render this in the template.

Comment (0xF013) [inline app/assets/javascripts/angular/directives/components/table-pagination-directive.js:71]: @myabc yes we do, I was just being careful around unknown codebase

**Pull request merged**